### DISCUSSION TITLE
SLS setup

### DISCUSSION THREAD

**Discussion by thinktank9** [nishi2go/mas-standalone-install#5] [Q&A]
first of all thank you for this repo and the LinkedIn post, really really helpful.

I am trying to set up sls but I am not able to get it to work. the script in this repo to install sls doesn't work for me so I install the operator manually from the OperatorHub and then run the rest of the script. everything goes through fine except for the last step where it gets stuck. The license service doesn't get initialized. I know for sure that I am putting in the correct entitlement key, I have done an installation from scratch 2 times, uninstalled sls completely and installed it a couple of times, cheked the entitlement key secret, couldn't find anything. The initialized status stays False with the following message - "The licensing system has not been initialized. Upload a valid entitlement file to enable the Token Pool and License Mgmt APIs."

![image](https://user-images.githubusercontent.com/14940852/130243284-833edf5b-53c8-4896-a73d-7d2509fca5a3.png)

I am not sure if it's the entitlement key that has the issue. I have access to "all" container software (see screenshot below), I don't know if I need a specific entitlement to sls or something on my account.

![image](https://user-images.githubusercontent.com/14940852/130242901-2bf674cc-fd11-415f-b295-c8f16d263535.png)

Not sure if the URLs that the license service is trying to ping are incorrect.

Any help would be appreciated.

**Comment by nishi2go** [comment]
Would you try to run the initial setup of MAS? It seems to be valid and it might clear the error when uploading a Token license file.
